models/optional_modules/consent/snowflake/snowplow_web_consent_log.sql

```diff
@@ -43,13 +45,15 @@ select
   p.page_view_id,
   p.domain_sessionid,
   p.derived_tstamp,
+  p.load_tstamp,
   p.event_name,
   p.event_type,
   p.basis_for_processing,
   p.consent_url,
   p.consent_version,
   array_to_string(p.consent_scopes, ', ') as consent_scopes,
   array_to_string(p.domains_applied, ', ') as domains_applied,
-  p.cmp_load_time
+  coalesce(p.gdpr_applies, false) as gdpr_applies,
+  round(p.cmp_load_time, 1) as cmp_load_time
```

Comment:
⛏️ Is this only recorded to 1 decimal place? Otherwise I would avoid rounding where possible outside of a reporting context.

Reply:
For general use was it does not necessarily make sense to interpret this in a lower value so tracking could have been enough with a less precise value IMHO and the original values come with a long list of decimals hence the conversion. Can remove this for any scientific analysis options though, as rounding was already place downstream.

Reply:
If we have a good idea of the true precision of the tracking and could round to that - that would be best. If we have no idea then suggest we leave it as raw.

Reply:
best to remove it at this stage